client/src/BasicLayout.js

```diff
@@ -112,7 +147,7 @@ export default class BasicLayout extends Component {
             />
             <span style={{ float: 'right', marginRight: 12 }}>
               <span style={{ color: '#87BC26', marginRight: 5, fontSize: 16 }}>
-                {loginModal ? '' : `Hello ${userName}`}
+                {loginModal ? '' : `Hello ${isAuthenticated.username}`}
```

Comment:
why isAuthenticated.username?

Reply:
e để thế vì e để isAuth là lấy toàn bộ data của 1 user

Reply:
thế thì biến loginInfo ko cần thiết? Nếu k cần thiết thì bỏ nhé

Reply:
đâu logininfo là để lấy data từ form lúc login mà